docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,97 @@
+  ``convert_model()`` returns an openvino.runtime.Model object which can be compiled and inferred or serialized to IR.
+
+.. code-block:: python
+
+   import torchvision
+   
+   model = torchvision.models.resnet50(pretrained=True)
+   ov_model = convert_model(model)
+
+The following types are supported as an input model for ``convert_model()``:
+
+* PyTorch - ``torch.nn.Module``, ``torch.jit.ScriptModule``, ``torch.jit.ScriptFunction``. Refer to the :doc:`Converting a PyTorch Model<openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch>` article for more details.
+* TensorFlow/ TensorFlow2 / Keras - ``tf.keras.Model``, ``tf.keras.layers.Layer``, ``tf.compat.v1.Graph``, ``tf.compat.v1.GraphDef``, ``tf.Module``, ``tf.function``, ``tf.compat.v1.session``, ``tf.train.checkpoint``. Refer to the :doc:`Converting a TensorFlow Model<openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow>` article for more details.
+
+``convert_model()`` accepts all parameters available in the MO command-line tool. Parameters can be specified by Python classes or string analogs, similar to the command-line tool.
+
+Example 1:
+
+.. code-block:: python
+
+   from openvino.runtime import PartialShape, Layout
+   
+   ov_model = convert_model(model, input_shape=PartialShape([1,3,100,100]), mean_values=[127, 127, 127], layout=Layout("NCHW"))
+
+Example 2:
+
+.. code-block:: python
+
+   ov_model = convert_model(model, input_shape="[1,3,100,100]", mean_values="[127,127,127]", layout="NCHW")
+
+
+The ``input`` parameter can be set by a ``tuple`` with a name, shape, and type. The input name of the type string is required in the tuple. The shape and type are optional.
+The shape can be a ``list`` or ``tuple`` of dimensions (``int`` or ``openvino.runtime.Dimension``), or ``openvino.runtime.PartialShape``, or ``openvino.runtime.Shape``. The type can be of numpy type or ``openvino.runtime.Type``.
+
+.. code-block:: python
+
+   ov_model = convert_model(model, input=("input_name", [3], np.float32))
+
+For complex cases, when a value needs to be set in the ``input`` parameter, the ``InputCutInfo`` class can be used. ``InputCutInfo`` accepts four parameters: ``name``, ``shape``, ``type``, and ``value``. 
+
+``InputCutInfo("input_name", [3], np.float32, [0.5, 2.1, 3.4])`` is equivalent of ``InputCutInfo(name="input_name", shape=[3], type=np.float32, value=[0.5, 2.1, 3.4])``.
+
+Supported types for ``InputCutInfo``:
+
+* name: ``string``.
+* shape: ``list`` or ``tuple`` of dimensions (``int`` or ``openvino.runtime.Dimension``), ``openvino.runtime.PartialShape``, ``openvino.runtime.Shape``.
+* type: ``numpy type``, ``openvino.runtime.Type``.
+* value: ``numpy.ndarray``, ``list`` of numeric values, ``bool``.
+
+.. code-block:: python
```

Comment:
Should we tell that this example demonstrating freezing `input_name` input to some value?
Also, is the command below is valid without listing other inputs that are not frozen?

Reply:
Added comments for all examples.